README.md

```diff
@@ -73,6 +73,11 @@ git clone https://github.com/tensorflow/models.git
 export PYTHONPATH=$PYTHONPATH:/path/to/models
 ```
 
+If you are using in a Windows environment, you may need to use the following command with PowerShell:
+```shell
+$env:PYTHONPATH += "C:\path\to\models"
```

Comment:
In order to be consistent, Can we remove "C" as shown below 
$env:PYTHONPATH += ":\path\to\models"